app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
```

Comment:
Code Style Issue: The variable names `comfort_class` and `clean_mark` are not descriptive enough. It's not clear what they stand for in the context of a Car class. Consider renaming them to something more descriptive, for example `comfort_level` and `cleanliness_level`.